diff --git a/src/fmripost_aroma/utils/utils.py b/src/fmripost_aroma/utils/utils.py
index 41240d0..a0fe5a2 100644
--- a/src/fmripost_aroma/utils/utils.py
+++ b/src/fmripost_aroma/utils/utils.py
@@ -9,7 +9,7 @@ import nibabel as nb
 import numpy as np
 import pandas as pd
 from nilearn import masking
-from nilearn._utils import load_niimg
+from nilearn._utils.niimg import load_niimg
 
 # Define criteria needed for classification (thresholds and
 # hyperplane-parameters)
