diff --git a/src/fmripost_aroma/workflows/aroma.py b/src/fmripost_aroma/workflows/aroma.py
index 918915a..01443bd 100644
--- a/src/fmripost_aroma/workflows/aroma.py
+++ b/src/fmripost_aroma/workflows/aroma.py
@@ -162,6 +162,7 @@ in the corresponding confounds file.
     calc_median_val = pe.Node(
         MedianValue(),
         name='calc_median_val',
+        mem_gb=mem_gb['resampled'],
     )
     workflow.connect([
         (inputnode, calc_median_val, [('bold_mask_std', 'mask_file')]),
@@ -171,6 +172,7 @@ in the corresponding confounds file.
     calc_bold_mean = pe.Node(
         MeanImage(),
         name='calc_bold_mean',
+        mem_gb=mem_gb['resampled'],
     )
     workflow.connect([
         (inputnode, calc_bold_mean, [('bold_mask_std', 'mask_file')]),
