diff --git a/sdcflows/interfaces/bspline.py b/sdcflows/interfaces/bspline.py
index f2b1dfaa..2f0199fe 100644
--- a/sdcflows/interfaces/bspline.py
+++ b/sdcflows/interfaces/bspline.py
@@ -309,12 +309,12 @@ class _ApplyCoeffsFieldInputSpec(BaseInterfaceInputSpec):
     fmap2data_xfm = InputMultiObject(
         File(exists=True),
         desc="the transform by which the target EPI can be resampled on the fieldmap's grid.",
-        xor="data2fmap_xfm",
+        xor=["data2fmap_xfm"],
     )
     data2fmap_xfm = InputMultiObject(
         File(exists=True),
         desc="the transform by which the fieldmap can be resampled on the target EPI's grid.",
-        xor="fmap2data_xfm",
+        xor=["fmap2data_xfm"],
     )
     in_xfms = traits.List(
         traits.List(traits.List(traits.Float)),
